en/camera_protocol.md

```diff
@@ -2,6 +2,31 @@
 
 The camera protocol allows to configure camera payloads and request their status. It supports photo and video cameras and includes messages to query and configure the onboard camera storage.
 
+## Camera Information
+
+### Components:
```

Comment:
IMO a heading should never have a colon terminating it. There are a bunch of these here.